### PR TITLE
fix(flow-item): Stack error in Edge legacy

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** None

## Summary

fix(flow-item): Stack error in Edge legacy

### CODE REVIEW

File: src/components/calcite-flow-item/resources.ts [hunk 1]
```diff
@@ -12,7 +12,8 @@ export const CSS = {
   menuButton: "menu-button",
   menu: "menu",
   menuOpen: "menu--open",
-  fabContainer: "fab-container"
+  fabContainer: "fab-container",
+  defaultSlotContainer: "default-slot-container"
```
Comment (jcfranco) [inline src/components/calcite-flow-item/resources.ts:16]: It doesn't seem like this class is being used.

File: src/components/calcite-flow-item/calcite-flow-item.tsx [hunk 2]
```diff
@@ -368,12 +372,20 @@ export class CalciteFlowItem {
...
-      <div class={CSS.fabContainer} slot={PANEL_SLOTS.fab}>
+      <div key="fab-container" class={CSS.fabContainer} slot={PANEL_SLOTS.fab}>
         <slot name={SLOTS.fab} />
       </div>
     ) : null;
   }
 
+  renderDefaultSlot(): VNode {
+    return (
+      <div key="default-slot-container" class={CSS.defaultSlotContainer}>
```
Comment (jcfranco) [inline src/components/calcite-flow-item/calcite-flow-item.tsx:383]: Just curious if `key` is needed here since this node is not rendered conditionally.

**Comment by jcfranco** [comment]
Was this only an issue in Edge legacy? Seems like this would've caused rendering trouble in other browsers because of the missing `key`s. Maybe it isn't common for a reused component to change part-toggling props.

**Comment by driskull** [comment]
@jcfranco I can remove the keys, its the div around the slot that fixed it.

File: src/components/calcite-flow-item/resources.ts (hunk 1 above)
Comment (driskull) [reply src/components/calcite-flow-item/resources.ts:16]: Yeah but Its better than a div without anything.

**Comment by driskull** [comment]
This might be another issue for Stencil. Putting a default slot inside another nested component could be the issue but would need to investigate more.

File: src/components/calcite-flow-item/resources.ts (hunk 1 above)
Comment (jcfranco) [reply src/components/calcite-flow-item/resources.ts:16]: Strong disagree. Let's not add anything that isn't being used.

File: src/components/calcite-flow-item/calcite-flow-item.tsx [hunk 3]
```diff
@@ -374,6 +374,15 @@ export class CalciteFlowItem {
     ) : null;
   }
 
+  renderDefaultSlot(): VNode {
+    return (
+      // default slot requires a container for edge legacy.
```
Comment (jcfranco) [inline src/components/calcite-flow-item/calcite-flow-item.tsx:379]: Nitpick: could you add more context about this being a workaround?

**Comment by driskull** [comment]
If I set `slotChildNodesFix: false` then this is no longer necessary. I'll open a Stencil issue instead of merging this right now.

**Comment by driskull** [comment]
closing for now. Will reopen if necessary.

**Pull request closed without merging**